modules/@apostrophecms/job/index.js

```diff
@@ -195,7 +196,10 @@ module.exports = {
           job = await self.start(options);
 
           const notification = await self.triggerNotification(req, 'progress', {
-            jobId: job._id
+            jobId: job._id,
+            ids: req.body._ids,
+            action: options.action,
+            moduleName: options.moduleName
```

Comment:
a lot of missing job info when starting with `run`

Reply:
`run` doesn't assume it's going to run on some batch of ids. It could be any long-running job. `runBatch` is a convenience specifically for when you know it's the same operation over and over.